superset/migrations/versions/5fd49410a97a_add_columns_for_external_management.py

```diff
@@ -0,0 +1,80 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Add columns for external management
+
+Revision ID: 5fd49410a97a
+Revises: c53bae8f08dd
+Create Date: 2022-01-19 07:34:20.594786
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "5fd49410a97a"
+down_revision = "c53bae8f08dd"
+
+import sqlalchemy as sa
+from alembic import op
+
+
+def upgrade():
+    with op.batch_alter_table("dashboards") as batch_op:
+        batch_op.add_column(
+            sa.Column("is_managed_externally", sa.Boolean(), nullable=True)
```

Comment:
Shouldn't `is_managed_externally` be required by default? Something like:

`sa.Column("is_managed_externally", sa.Boolean(), nullable=False, default=False)`

Reply:
Good point... I'm not sure why Alembic didn't add `default=False`, since it's defined [in the model](https://github.com/apache/superset/pull/18093/files#diff-e4b142ee2a4f8ad0bc169ea59bff5bd5eed4ea2570870381f9e3efbae133783fR95). Let me add it manually, thanks!

Reply:
Fixed it by using `server_default`. Now the new column is populated with `false` values, and the same happens when a new resource is created.